resources/views/pages/user/[user]/game/[game]/compare.blade.php

```diff
@@ -74,7 +74,7 @@
 
 @if ($canModerate)
     <x-hidden-controls>
-        <div style="width:20%">
+        <div class="grid md:grid-cols-3 lg:grid-cols-4 xl:grid-cols-5">
```

Comment:
This div was the root cause of the issue.

Rather than a hardcoded width, it's better to reach for a grid layout with specified column counts at certain breakpoints (generally XS and SM just have 1 column, which is the default).